.pre-commit-config.yaml

```diff
@@ -7,16 +7,16 @@ repos:
       - id: check-yaml
 
   - repo: https://github.com/PyCQA/isort
-    rev: 5.8.0
+    rev: 5.9.3
     hooks:
       - id: isort
 
   - repo: https://github.com/python/black
-    rev: 21.5b1
+    rev: 21.7b0
     hooks:
       - id: black
   - repo: https://github.com/keewis/blackdoc
-    rev: v0.3.3
+    rev: v0.3.4
     hooks:
       - id: blackdoc
   - repo: https://gitlab.com/pycqa/flake8
```

Comment:
This was changed by `pre-commit autoupdate`, which I had to do in order to silence an error

```python
Traceback (most recent call last):
  File "/home/tom/.cache/pre-commit/repo03g1je45/py_env-python3/bin/blackdoc", line 5, in <module>
    from blackdoc.__main__ import main
  File "/home/tom/.cache/pre-commit/repo03g1je45/py_env-python3/lib/python3.8/site-packages/blackdoc/__main__.py", line 12, in <module>
    from .blackcompat import (
  File "/home/tom/.cache/pre-commit/repo03g1je45/py_env-python3/lib/python3.8/site-packages/blackdoc/blackcompat.py", line 9, in <module>
    import toml
ModuleNotFoundError: No module named 'toml'
```
which appeared when trying to work from a new PC. I think this is the same as https://github.com/keewis/blackdoc/issues/100, and possibly should have its own issue.